src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_CAR = 40;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  const totalRent = RENT_CAR * days;
```

Comment:
```suggestion
  const basicPrice = RENT_CAR * days;
```
Same here. It's not a total price until discounts are counted